specs/core/1_custody-game.md

```diff
@@ -624,20 +637,21 @@ def process_bit_challenge_response(state: BeaconState,
     assert verify_merkle_branch(
         leaf=hash_tree_root(response.chunk),
         branch=response.data_branch,
-        depth=math.log2(next_power_of_two(challenge.chunk_count)),
+        depth=math.ceil(math.log2(challenge.chunk_count)),
         index=response.chunk_index,
         root=challenge.data_root,
     )
     # Verify the chunk bit leaf matches the challenge data
     assert verify_merkle_branch(
         leaf=response.chunk_bits_leaf,
         branch=response.chunk_bits_branch,
-        depth=math.log2(next_power_of_two(challenge.chunk_count) // 256),
+        depth=math.ceil(math.log2(challenge.chunk_count)) >> 8,
```

Comment:
I did this in a rush, can someone check my logic here please.

Reply:
Looks correct. See above for my comment how to remove the FP arithmetic